src/agent/docker-rest-agent/intergration-test/test.py

```diff
@@ -6,18 +6,17 @@
 client = docker.from_env()
 
 with open('msp.zip', 'rb') as node_msp, open('tls.zip', 'rb') as tls, open('block.zip', 'rb') as block, open('peer_config.zip', 'rb') as peer_config, open('orderer_config.zip', 'rb') as orderer_config:
-
     data = {
                 'msp':base64.b64encode(node_msp.read()),
                 'tls':base64.b64encode(tls.read()),
                 'bootstrap_block':base64.b64encode(block.read()),
                 'peer_config_file':base64.b64encode(peer_config.read()),
                 'orderer_config_file':base64.b64encode(orderer_config.read()),
-                'img': 'yeasy/hyperledger-fabric:2.2.0',
+                # 'img': 'yeasy/hyperledger-fabric:2.2.0',
```

Comment:
@yeasy Do I need to delete this line or can I leave for a reference?

Reply:
Better change it to `hyperledger/cello-hlf:2.2.0`.